.github/workflows/reusable_push_pr.yaml

```diff
@@ -0,0 +1,116 @@
+name: Push/PR no fake prereleases
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        required: true
+        type: string
+      repo_full_name:
+        required: false
+        type: string
+      tag:
+        required: false
+        type: string
```

Comment:
do you know why it is needed? can we have as default v0.0.0?

Reply:
Yes, we can have it set as an env var always with the value "v.0.0.0"